cmd/svcat/testdata/output/completion-bash.txt

```diff
@@ -0,0 +1,277 @@
+# bash completion for completion                           -*- shell-script -*-
```

Comment:
This is why I wanted a "positive" test case. This file doesn't contain any of the svcat commands, and when I built the pr and ran it, nothing autocompleted. 😀 

```bash
make svcat
source <(./bin/svcat/svcat completion bash)
#svcat desTAB should autocomplete to svcat describe
```

I tweaked your code to use `cmd.Root().GenCompletion()` and that fixed it. Once you make that change, I think we are all set. Though please give it a try locally yourself as well.

Reply:
You're right! I just tried it on my machine and it isnt working. I know I tested before I committed, but I can't see what would have changed between then and now to make it behave differently (my kingdom for a time machine). 

The only thing I can think of is I hit `svcat comTAB`, it autocompleted to `svact completion/` (I was working in the svcat directory), and I mistakenly thought it had sucessfully performed a bash completion operation when it had just picked up the 'completion' directory.

I've made your suggested changes and can confirm `svcat tab` prompts with appropriate options.